BackendWebApp/src/test/java/uk/ac/ebi/biostd/webapp/application/rest/controllers/FileManagerResourceTest.java

```diff
@@ -110,12 +122,13 @@
     @MockBean
     private FileManagerService fileManagerService;
 
+    private File archive;
```

Comment:
this test is getting to complex IMHO, I think we can splited by creating integration test by end point, IMHO there is not necesary to have a one by one mapping beetween classes and test.

Reply:
You're right, but the only reason why I added this was because of this line which you made me realize was not necessary:
https://github.com/EBIBioStudies/BioStd-Backend/blob/159493946/show-archives/BackendWebApp/src/main/java/uk/ac/ebi/biostd/webapp/application/rest/controllers/FileManagerResource.java#L187

By removing this, we won't no longer need a fake file system and the tests can be managed with plain mock files. I'll add this change and we can review if still need to split this. But I do agree with you that it would be good to have some integration testing. We can discuss this later :smile: